JSDemos/Demos/DateBox/Overview/description.md

```diff
@@ -1,5 +1,25 @@
-The **DateBox** is&nbsp;a&nbsp;UI component that displays the date and time in&nbsp;one of&nbsp;DevExtreme [predefined formats][0] or&nbsp;a&nbsp;format defined in&nbsp;an&nbsp;external localization library, or&nbsp;a&nbsp;type in&nbsp;the required date/time value. In&nbsp;this demo, the &laquo;Custom format&raquo; **DateBox**&rsquo;es value is&nbsp;formatted [using the Globalize library][1]. Alternatively, you can [use Intl][2].
+The [DateBox](/Documentation/ApiReference/UI_Widgets/dxDateBox/) is an editor that allows users to select date and time and displays them. This demo illustrates the following **DateBox** properties:
```

Comment:
If possible, I would always try to avoid definitions, i.e. "This is control is a control that allows you to do smth...". A lot is obvious based on the control name itself. But that's not as important as the sentence structure... The X is Y that allows you to Z - this sounds too formal (people already criticize our language for being too "boring, academic"). This type of sentence could probably work if you need to introduce a complicated term - but we actually try to make it all sound nice and easy. 

```suggestion
The [DateBox](/Documentation/ApiReference/UI_Widgets/dxDateBox/) editor helps users enter or modify date and time values. This demo illustrates the following **DateBox** properties:
```